plugins/metrics/scripts/send_session_metrics.py

```
@@ -0,0 +1,344 @@
+    """
+    try:
+        id_file = metrics_dir / ANONYMOUS_ID_FILE_NAME
+
+        if id_file.exists():
+            return id_file.read_text().strip()
+
+        # Generate new random UUID
+        import uuid
+        anonymous_id = str(uuid.uuid4())
+        id_file.write_text(anonymous_id)
+        return anonymous_id
```

Ruff (0.14.4)

Consider moving this statement to an `else` block

(TRY300)